### ISSUE TITLE
About Bigquery

### ISSUE THREAD

**Issue by ZacharyChenpk** [willprice221/ethereum_clustering#1]
Ummm, it seemed that I have no access to query tables referred in functions.py and I cannot find it in Google BigQuery Public Database, so how can I fetch the data?

**Comment by mgsgde** [comment]
"bigquery-public-data.crypto_ethereum" is the BigQuery dataset name of the public ethereum dataset. (Link: https://bigquery.cloud.google.com/dataset/bigquery-public-data:crypto_ethereum)

**Comment by mgsgde** [comment]
@willprice221 Could you please provide your sql commands for generating all the tables you created in eth-tokens.alldata? I suppose you have used the aforementioned public ethereum dataset from bigquery to generate your tables in eth-tokens.alldata?

**Comment by ashishde1991** [comment]
@willprice221  , Yes I tried Finding the eth-tokens.alldata in BIG Query , It Aint there , 
Will Could you Please help us out with the procedure to create the table eth-tokens ?
